core/src/views/Profile.vue

```diff
@@ -255,6 +255,12 @@ export default {
 			// For some reason the returned string has prepended whitespace
 			return getComputedStyle(document.body).getPropertyValue('--color-main-background').trim()
 		},
+
+		emptyProfileMessage() {
+			return this.isCurrentUser
+				? t('core', 'You haven\'t added any info yet')
+				: (this.displayname || this.userId) + ' ' + t('core', 'hasn\'t added any info yet')
```

Comment:
Building sentences like that might not work in all languages, better include the name as a parameter in the translatable text.
```suggestion
				:  + t('core', '{user} hasn\'t added any info yet', { user: (this.displayname || this.userId) })
```

Reply:
Done!